3.2/root/usr/share/container-scripts/mongodb/init-petset-replset.sh

```diff
@@ -107,11 +107,17 @@ function add_member() {
   "
 
   # TODO: replace this with a call to `replset_addr` from common.sh, once it returns host names.
-  local replset_addr
-  replset_addr="${MONGODB_REPLICA_NAME}/$(find_endpoints | paste -s -d,)"
+  local endpoints
+  endpoints="$(find_endpoints | paste -s -d,)"
+
+  if [ -z "${endpoints}" ]; then
+    info "ERROR: couldn't join replica set!"
+    info "CAUSE: couldn't find hosts that are belongs to the replica set."
```

Comment:
Suggestion:

```
"CAUSE: DNS lookup for '${MONGODB_SERVICE_NAME:-mongodb}' returned no results."
```

Otherwise, "that are belongs" needs fixing.